feature/client-api/src/main/java/com/simprints/feature/clientapi/mappers/response/LibSimprintsResponseMapper.kt

```diff
@@ -64,6 +112,16 @@ internal class LibSimprintsResponseMapper @Inject constructor() {
         )
     }
 
+    private inline fun Bundle.appendDataForVersion(
+        response: ActionResponse,
```

Comment:
TBH, I find it a little confusing that appendDataForVersion() does not take version as a parameter but actually returns it.

Reply:
Changed to `appendDataPerVersion` which is a bit more precise.